Dynatrace/dynatrace-operator
# Simplify logging and get rid of log singleton

# Description
This PR is a preparation for running the troubleshoot command within the support archive command.
First, it gets rid of the hidden log singleton in troubleshoot. This is needed so that the support archive command can provide an own logger to grab all output into a file. With this, logging had to be reworked a bit to keep separate subtest loggers with dedicated names.

Furthermore, formatting log lines seems to be implemented overly complicated, this has been simplified along the way.

## How can this be tested?
Build the operator, deploy it and run the troubleshoot command. Everything should work as before.

## Checklist
